modern/src/map/GeofenceMap.js

```diff
@@ -4,6 +4,15 @@ import { useEffect, useState } from 'react';
 import { map } from './Map';
 import { useEffectAsync } from '../reactHelper';
 import { reverseCoordinates } from './mapUtil';
+import circle from '@turf/circle';
+
+
+const circleToPolygon = (item) => {
+  let coord = item.area.replace(/CIRCLE|\(|\)|,/g, " ").trim().split(/ +/);
+  var options = {steps: 64, units: 'meters'};
```

Comment:
```suggestion
  var options = { steps: 64, units: 'meters' };
```

I would also reduce the number of steps.

Reply:
ok, steps is now 32